### ISSUE TITLE
Accessibility Issue - v12.2.0 - Recycle Bin button does not have an accessible name

### ISSUE THREAD

**Issue by Truealex4** [umbraco/Umbraco-CMS#15071]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

v10.6.1 and v12.2.0

### Bug summary

If I run the lighthouse accessibility report on the Recycle Bin page then it flags an issue with the checkbox that is used to 'select all' - there is no accessibile name so a person using a screenreader etc would not be told what the button does etc.

### Specifics

/umbraco#/content/content/recyclebin

This is not browser specific

### Steps to reproduce

STEPS
1. Access a site on v10.6.1 or v12.2.0
2. Login with valid credentials
3. Select the 'Recycle Bin' node from the 'Content' CMS
4. Run the Google lighthouse accessibility report

### Expected result / actual result

**EXPECTED RESULT** - No issues found
**ACTUAL RESULT** - Issue identified wirth the 'select all' CTA not having an accessible name as shown below:

![image](https://github.com/umbraco/Umbraco-CMS/assets/98405431/c071dc7d-1e68-47a4-becb-97d823e86f9c)

**Failing element:**
div.umb-table-head > div.umb-table-row > div.umb-table-cell > button.umb-table__action
<button type="button" class="umb-table__action" ng-show="vm.allowSelectAll" ng-click="vm.selectAll()" aria-hidden="false">

**Comment by andr317c** [comment]
Hey! Thanks for posting this issue! I was able to reproduce this accessibility issue on versions 10.7.0 and 12.2.0. A few reproduction steps were missing:
1. Create a DocumentType
2. Create A ContentNode
3. Delete the ContentNode
4. Go into the Content RecycleBin (You might need to reload)
5. Change the view from Grid to List
6. Run the Google Lighthouse accessibility report

This issue is something we would like help with, I'll set this issue up for grabs!😄

**Comment by mikecp** [comment]
Closing as per merging of #15118
